File: problems/problems_3575/Solution.java
package problems.problems_3575;

import com.alibaba.fastjson.JSON;
import java.util.*;
import qubhjava.BaseSolution;


public class Solution extends BaseSolution {
    public int goodSubtreeSum(int[] vals, int[] par) {
        
    }

    @Override
    public Object solve(String[] inputJsonValues) {
        int[] vals = jsonArrayToIntArray(inputJsonValues[0]);
		int[] par = jsonArrayToIntArray(inputJsonValues[1]);
        return JSON.toJSON(goodSubtreeSum(vals, par));
    }
}

File: problems/problems_3575/Solution.cpp
//go:build ignore
#include "cpp/common/Solution.h"


using namespace std;
using json = nlohmann::json;

class Solution {
public:
    int goodSubtreeSum(vector<int>& vals, vector<int>& par) {
        
    }
};

json leetcode::qubh::Solve(string input_json_values) {
	vector<string> inputArray;
	size_t pos = input_json_values.find('\n');
	while (pos != string::npos) {
		inputArray.push_back(input_json_values.substr(0, pos));
		input_json_values = input_json_values.substr(pos + 1);
		pos = input_json_values.find('\n');
	}
	inputArray.push_back(input_json_values);

	Solution solution;
	vector<int> vals = json::parse(inputArray.at(0));
	vector<int> par = json::parse(inputArray.at(1));
	return solution.goodSubtreeSum(vals, par);
}
